packages/vscode-extension/src/handlers.ts

```diff
@@ -913,73 +913,6 @@ function isLoginFailureError(error: FxError): boolean {
   return !!error.message && error.message.includes("Cannot get user login information");
 }
 
-function showWarningMessageWithProvisionButton(message: string): void {
-  window
-    .showWarningMessage(message, localize("teamstoolkit.handlers.provisionResourcesButton"))
```

Comment:
If string is not used any more, could you please also remove it in `package.nls.json`? Same as other localized strings.

Reply:
Updated, thx!